packages/editor/src/extentions/ListItem.ts

```diff
@@ -0,0 +1,8 @@
+import { ListItem } from '@tiptap/extension-list';
+
+export const CustomListItem = ListItem.extend({
+  name: 'listItem',
+  // TODO: bummer, content does not allow mix of inline and block, so can't allow both text and nested lists
+  // Documentation: https://tiptap.dev/docs/editor/core-concepts/schema#nodes-and-marks
+  content: 'inline*',
```

Comment:
So this wouldn't work? I would expect that the inline would be a node just as much a list is a node.


```suggestion
  content: '(inline|list?)*',
```

From the docs:

> ```ts
> Node.create({
>   // can have one or more paragraphs, or lists (if lists are used)
>   content: '(paragraph|list?)+',
> })
> ```

And what if you would specify that a list item needs to have a paragraph as the child. Wouldn't that solve it?

Reply:
`Uncaught (in promise) SyntaxError: Mixing inline and block content (in content expression '(inline|list?)*') (at index.js:1797:22)`

Het is een bekend 'probleem' van prosemirror. Ik zal uiteindelijk moeten toestaan dat elk `<li>` element dus een `<p>` als child heeft. `<li>voorbeeld</li>` wordt dus altijd `<li><p>voorbeeld</p></li>`. 

Je moet dus dus die p-tag accepteren mocht je bijv. afbeeldingen of lijsten willen kunnen nesten. `<li><ul><li>voorbeeld</li></ul></li>`. 

Ik markeer dit even als probleem voor later